projects/core/src/i18n/translation-chunk.service.ts

```diff
@@ -18,10 +20,28 @@ export class TranslationChunkService {
   }
 
   private getChunkFromConfig(mainKey: string): string {
```

Comment:
This function is called every time when we get translations. It would be good to optimize it (not interate over whole config every time).

I think we can in the constructor transform the config locally back to the shape:
```
[mainKey: string]: string // chunk name
```
and use this local optimized config in the `getChunkFromConfig` method.

By the way, while building this optimized local config, we can eaisly detect duplicates and warn specifically about them.